apisix/cli/ops.lua

```diff
@@ -190,6 +190,15 @@ local function init(env)
         checked_admin_key = true
     end
 
+    -- check if APISIX_BYPASS_ADMIN_API_AUTH=true
+    local allow_none_auth = getenv("APISIX_BYPASS_ADMIN_API_AUTH")
+    if allow_none_auth == "true" then
+        checked_admin_key = true
+        print("Warning! AdminKey is bypassed because of APISIX_BYPASS_ADMIN_API_AUTH=true.",
+            "If you are deploying APISIX in a production environment,",
+            "please disable it and set a secure password for the adminKey!")
+    end
+
```

Comment:
Why are there duplicate codes?

Reply:
Do you mean that hint appears in both stdout and log?

1. `opt.lua` checks if allow_admin in config.yaml is correct before APISIX starts. We need to bypass the adminKey check here.
2. `init.lua` checks the adminKey when accessing the Admin API. We need to bypass check_token, too.

That's why I have implemented bypass-logic in two different stage.(before and after the start).

Reply:
In other words, opt.lua does a static check on the configuration file (if it is not 127.0.0.0/24 and the adminKey is empty, APISIX will not start). 

We also have to bypass the checks here.

Reply:
If there is repeated code, then it needs to be abstracted into a function